fonts.css

```diff
@@ -0,0 +1,7 @@
+@font-face {
+  font-family: 'Montserrat';
+  src: url('Montserrat.woff2') format('woff2'),
+       url('Montserrat.woff') format('woff');
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/TESLA-Website-NataSG/pullRequest?prid=6083130)